tests/integration/dom0-template/var/lib/net-snmp/snmpd.conf

```diff
@@ -0,0 +1,55 @@
+#
+# net-snmp (or ucd-snmp) persistent data file.
+#
+############################################################################
+# STOP STOP STOP STOP STOP STOP STOP STOP STOP 
+#
+#          **** DO NOT EDIT THIS FILE ****
+#
+# STOP STOP STOP STOP STOP STOP STOP STOP STOP 
+############################################################################
+#
+# DO NOT STORE CONFIGURATION ENTRIES HERE.
+# Please save normal configuration tokens for snmpd in /etc/snmp/snmpd.conf.
+# Only "createUser" tokens should be placed here by snmpd administrators.
+# (Did I mention: do not edit this file?)
+#
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
```suggestion
```

Can the consecutive empty lines be removed?

Reply:
This file is auto-generated by running `systemctl start snmpd`. These spaces and empty lines are generated by it.
But it's OK to remove these spaces and empty lines. I have already removed them.